metricbeat/helper/interfacer.go

```diff
@@ -32,6 +33,7 @@ type ModuleConfig struct {
 	MetricSets []string `config:"metricsets"`
 	Enabled    bool     `config:"enabled"`
 	Selectors  []string `config:"selectors"`
+	Filter     []filter.FilterConfig
```

Comment:
I think is more efficient to have the filters per module instead of filters per Beat as you can execute the filters only on the data exported by the module and not all data.